Jenkinsfile

```diff
@@ -520,16 +522,48 @@ def cleanDir(path){
 }
 
 def dotnet(Closure body){
-  def dockerTagName = 'docker.elastic.co/observability-ci/apm-agent-dotnet-sdk-linux:latest'
-  sh label: 'Docker build', script: "docker build --tag ${dockerTagName} .ci/docker/sdk-linux"
+
   def homePath = "${env.WORKSPACE}/${env.BASE_DIR}"
-  docker.image("${dockerTagName}").inside("-e HOME='${homePath}' -v /var/run/docker.sock:/var/run/docker.sock"){
+  withEnv([
+    "HOME=${homePath}",
+    "DOTNET_ROOT=${homePath}/.dotnet",
+    "PATH+DOTNET=${homePath}/.dotnet/tools:${homePath}/.dotnet"
+    ]){
+    sh(label: 'Install dotnet SDK', script: """
+    mkdir -p \${DOTNET_ROOT}
+    # Download .Net SDK installer script
+    curl -s -O -L https://dotnet.microsoft.com/download/dotnet-core/scripts/v1/dotnet-install.sh
```

Comment:
After the Solarwinds fiasco, I wonder if we should store a copy of this script ourselves instead of downloading it directly each time. WDYT?

Reply:
yep, we have to start consuming all those "binaries" from a bucket we own

Reply:
The script determines which tarball to download based on the OS, platform and version number passed, constructing a URL to `https://aka.ms/dotnet`. I guess we would also need to download the resulting tarballs and also place in a bucket?